app/src/main/java/com/infomaniak/mail/views/AvatarView.kt

```diff
@@ -219,6 +234,13 @@ class AvatarView @JvmOverloads constructor(
         }
     }
 
+    private data class InternalState(var correspondent: Correspondent?, var bimi: Bimi?) {
```

Comment:
I prefer : 
```kotlin    
private data class InternalState(var correspondent: Correspondent? = null, var bimi: Bimi? = null)
```

especially because you're not doing anything in the `setInternalState` method. Also, it's a bit ugly to initialize `initialState` with `InitialState(null, null)`.

Finally, the name is a bit too wide for me.

If other are ok with the current state, I'm also ok.